tests/integration/targets/azure_rm_containerinstance/tasks/main.yml

```diff
@@ -143,6 +143,95 @@
       - output.containerinstances[1]['ports'] != None
       - output.containerinstances[1]['containers'] != None
 
+- name: Create sample container instance with volume
+  azure_rm_containerinstance:
+    resource_group: "{{ resource_group }}"
+    name: myContainerInstanceGroup
+    os_type: linux
+    ip_address: public
+    location: eastus
+    ports:
+      - 80
+    containers:
+      - name: mycontainer1
+        image: httpd
+        memory: 1.5
+        volume_mounts:
+          - name: "aci_fileshare_volume{{ resource_group | hash('md5') | truncate(7, True, '') }}"
+            mount_path: "/data/files"
+        ports:
+          - 80
+          - 81
+    volumes:
+      - name: "aci_fileshare_volume{{ resource_group | hash('md5') | truncate(7, True, '') }}"
```

Comment:
The volume name must match the regex '[a-z0-9]([-a-z0-9]*[a-z0-9])?' (e.g. 'my-name')

Reply:
Hi @Fred-sun Thank you, Updated volume name.